scripts/init-project.sh

```
@@ -1,65 +1,85 @@
 set -e
 
-# Parámetros
-PROJECT_NAME="$1"
-VERSION="$2"
-DOCKER_USER="${3:-pablonicolas87}"
+# Parámetros\nPROJECT\_NAME="\$1"
 
-if [ -z "$PROJECT_NAME" ] || [ -z "$VERSION" ]; then
-  echo "Uso: init-project.sh <project-name> <version> [docker-user]"
-  exit 1
+VERSION="\$2"
```

Shellcheck (0.10.0)

[warning] VERSION appears unused. Verify use (or export if used externally).

(SC2034)